docker-compose.yml

```diff
@@ -181,13 +181,13 @@ sonar:
   environment:
     - "dependency:container==sonar-mysql"
     - "LDAP_URL=ldap://ldap:389"
```

Comment:
@mihail-dev Found this place where you could also use ${LDAP_SERVER} variable right?